index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Tribute Page</title>
+
```

Comment:
Add head elements that contain more info about the page.
It would be better for a web browser to understand your website.

Reply:
This is a good example, you're suggesting a change, and are giving a reason, kudos for this one! 💯 👏🏼